runtime/runtime/src/actions.rs

```diff
@@ -436,6 +438,11 @@ pub(crate) fn action_deploy_contract(
     );
     account.set_code_hash(code.get_hash());
     set_code(state_update, account_id.clone(), &code);
+    // Precompile the contract and store result (compiled code or error) in the database.
```

Comment:
Do we need to gate this behind some kind of runtime or compile-time flag? In particular, is it OK to enable this withoud adjusting the costs? 

I don't know the answer here, I guess that's a question for @bowenwang1996?

Reply:
Good point. This should come together with the change to the cost of deploy action

Reply:
I guess practically we shall recompute the costs along with other recent modifications and enabling Wasmer1.

Reply:
But I can comment it out, if you feel it dangerous, and submit everything else.